build-system/test-configs/test-overrides.json

```diff
@@ -0,0 +1,5 @@
+{
+  "SKIP": {},
+  "LARGE": {},
+  "XLARGE": {}
+}
```

Comment:
There are no tests currently in these categories because tests are only validated when they are added or modified.